packages/react-ui/src/components/molecules/Alert.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Alert as MuiAlert, Fade, Box, styled } from '@mui/material';
+import Typography from '../atoms/Typography';
+import AutoAwesomeOutlinedIcon from '@mui/icons-material/AutoAwesomeOutlined';
+
+const StyledAlert = styled(MuiAlert, {
+  shouldForwardProp: (prop) => prop !== 'isNeutral'
+})(({ isNeutral, theme }) => ({
+  borderRadius: theme.spacing(1),
+  paddingLeft: theme.spacing(1.5),
+  alignSelf: 'start',
+  width: '100%',
+  ...(isNeutral
+    ? {
+        backgroundColor: theme.palette.background.default,
+        color: theme.palette.text.primary
+      }
+    : {}),
+  '.MuiAlert-message': {
+    paddingTop: theme.spacing(0.75),
+    paddingBottom: theme.spacing(0.75),
+    width: '100%'
+  },
+  '.MuiAlert-icon': {
+    paddingTop: theme.spacing(1),
+    marginRight: theme.spacing(1),
+    svg: {
+      width: theme.spacing(2),
+      height: theme.spacing(2)
+    }
+  },
+  '.MuiAlert-action': {
+    paddingLeft: theme.spacing(1)
+  }
+}));
+
+const NeutralIcon = styled(AutoAwesomeOutlinedIcon)(({ theme }) => ({
+  color: theme.palette.warning.light
+}));
+
+const Alert = ({
+  title,
+  severity,
+  layout,
+  children,
+  onClose,
+  action,
+  icon,
+  ...otherProps
+}) => {
+  const [open, setOpen] = useState(true);
```

Comment:
I truly believe that open should be controlled by user.

Reply:
Agreed. I added an `open` prop to optionnally control that from the outside.